test/jdk/java/foreign/NativeTestHelper.java

```diff
@@ -123,6 +123,25 @@ public static boolean isPointer(MemoryLayout layout) {
      */
     public static final AddressLayout C_POINTER = ValueLayout.ADDRESS
             .withTargetLayout(MemoryLayout.sequenceLayout(Long.MAX_VALUE, C_CHAR));
+    /**
+     * The layout for the {@code size_t} C type
+     */
+    public static final ValueLayout C_SIZE_T = ValueLayout.ADDRESS.byteSize() == 8 ? C_LONG_LONG : C_INT;
+
+    // Common layout shared by some tests
+    // struct S_PDI { void* p0; double p1; int p2; };
+    public static final MemoryLayout S_PDI_LAYOUT = switch ((int) ValueLayout.ADDRESS.byteSize()) {
```

Comment:
I don't see differences between the two layouts created in the two cases?

Reply:
The first one has 4 extra bytes of padding

Reply:
Doh - I was looking for padding, but was distracted by the capitalization of the various constants :-)